caesar-cipher/caesar-cipher.c

```diff
@@ -0,0 +1,45 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int main(){
+    int n; 
+    scanf("Enter Input String Length: %d",&n);
```

Comment:
Please check your code.you can print statement while reading in c language.
Please 
https://stackoverflow.com/questions/12445474/how-to-print-using-scanf-function-in-c-programming

Reply:
Oops. My bad.